cirq-core/cirq/ops/projector.py

```diff
@@ -0,0 +1,161 @@
+import itertools
+from typing import (
+    Any,
+    Dict,
+    Iterable,
+    List,
+    Mapping,
+    Optional,
+    Union,
+)
+
+import numpy as np
+from scipy.sparse import coo_matrix
```

Comment:
Did `coo` perform better than `csr` or other types here ?

Reply:
Changed to `csr`.